security/authorization/engine/engine_test.go

```diff
@@ -23,10 +23,14 @@ import (
 
 	pb "github.com/envoyproxy/go-control-plane/envoy/config/rbac/v2"
 	cel "github.com/google/cel-go/cel"
+	"github.com/google/cel-go/checker/decls"
```

Comment:
For the line above, no need to give it a cel name, since there is no another import named "cel".
Same throughout all the files(in engine.go I also saw some places doing this)